tests/operator_test.go

```diff
@@ -53,381 +53,408 @@ import (
 	"kubevirt.io/kubevirt/tests"
```

Comment:
note: This file diff on github does not look good at all. Locally, with `kdiff3` is looks much better and easy to follow.